sockets/lib/commands.go

```diff
@@ -0,0 +1,103 @@
+/**
+ * Commands
+ * https://pokemonshowdown.com/
+ *
+ * Commands are an abstraction over IPC messages sent to and received from the
+ * parent process. Each message follows a specific syntax: a one character
+ * token, followed by any number of parametres separated by newlines. Commands
+ * give the multiplexer and IPC connection a simple way to determine which
+ * struct it's meant to be handled by, before enqueueing it to be distributed
+ * to workers to finally process their payload concurrently.
+ */
+
+package sockets
+
+import "strings"
+
+// IPC message types
+const SOCKET_CONNECT string = "*"
+const SOCKET_DISCONNECT string = "!"
+const SOCKET_RECEIVE string = "<"
+const SOCKET_SEND string = ">"
+const CHANNEL_ADD string = "+"
+const CHANNEL_REMOVE string = "-"
+const CHANNEL_BROADCAST string = "#"
+const SUBCHANNEL_MOVE string = "."
+const SUBCHANNEL_BROADCAST string = ":"
```

Comment:
Originally I chose `string` over `byte` so I wouldn't have to cast command tokens to `string` in `Command.Message`, but looking over the other code dealing with these, it makes no difference which type it is. It'd make more sense for them to be `byte` instead